features/io.asgardio.java.saml.sdk.server.feature/pom.xml

```diff
@@ -19,23 +19,23 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
     <parent>
-        <groupId>org.wso2.carbon.identity.agent.sso.java</groupId>
-        <artifactId>identity-agent-sso</artifactId>
+        <groupId>io.asgardio.java.saml.sdk</groupId>
+        <artifactId>java-saml-sdk</artifactId>
         <relativePath>../../pom.xml</relativePath>
         <version>5.5.9-SNAPSHOT</version>
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>org.wso2.carbon.identity.sso.agent.server.feature</artifactId>
+    <artifactId>io.asgardio.java.saml.sdk.server.feature</artifactId>
     <packaging>pom</packaging>
-    <name>SSO Agent server feature</name>
-    <url>http://wso2.org</url>
-    <description>This feature contains sso agent which is a library for sso and openid</description>
+    <name>Java SAML SDK server feature</name>
```

Comment:
Use same naming format. ie. Lets use title case for <name> in every pom.
"Java SAML SDK Server Feature"

Reply:
Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/2/commits/ccb42095df08cb3685ad47ae534f9dd826d4670e